### DISCUSSION TITLE
Best Performances (sometimes) not showing any correct plays

### DISCUSSION THREAD

**Discussion by Wypixla1** [ppy/osu#26874] [Q&A]
### Type

Other

### Bug description

Basically, I just simply went onto mrekk's profile for who knows why, and found that the best performances was completely wrong. 

### Screenshots or videos

<img width="779" alt="Screenshot 2024-01-30 182031" src="https://github.com/ppy/osu/assets/130538048/20949fb3-99a0-4e83-84e4-b32be3095f4f">

### Version

2024.130.2

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14105741/compressed-logs.zip)

**Comment by bdach** [comment]
Can't reproduce:

![osu_2024-01-31_09-35-31](https://github.com/ppy/osu/assets/20418176/a02ed14b-7ffd-4adf-811c-8aa52f0b7fac)
